ui/src/components/homepage/goodFridayTheme.js

```diff
@@ -0,0 +1,16 @@
+import { extendTheme } from '@chakra-ui/react';
```

Comment:
this file/theme is unused. also already Easter theme to handle the fonts for Easter/GoodFriday